include/fmt/ranges.h

```diff
@@ -604,6 +605,30 @@ struct formatter<
   }
 };
 
+// A string formatter.
+template <typename R, typename Char>
+struct formatter<
+    R, Char,
+    enable_if_t<range_format_kind<R, Char>::value == range_format::string>> {
+ private:
+  using range_type = detail::maybe_const_range<R>;
+  formatter<detail::std_string_view<Char>, Char> range_formatter_;
+
+ public:
+  FMT_CONSTEXPR formatter() {}
+
+  template <typename ParseContext>
+  FMT_CONSTEXPR auto parse(ParseContext& ctx) -> decltype(ctx.begin()) {
+    return range_formatter_.parse(ctx);
+  }
+
+  template <typename FormatContext>
+  auto format(range_type& range, FormatContext& ctx) const
+      -> decltype(ctx.out()) {
+    return range_formatter_.format(detail::std_string_view<Char>(range), ctx);
```

Comment:
This is inconsistent with the standard formatter: https://eel.is/c++draft/format.range.fmtstr. The range should be formatted as a string without conversion to a `string_view`.

Reply:
Is there a `set_brackets` without a bracket char?

Reply:
`set_brackets` can take empty string views if that's what you are asking.